server/tsuserver.py

```diff
@@ -105,24 +112,58 @@ def get_player_count(self):
         return len(self.client_manager.clients)
 
     def load_config(self):
-        with open('config/config.yaml', 'r') as cfg:
+        with open('config/config.yaml', 'r', encoding = 'utf-8') as cfg:
```

Comment:
small caveat; most people use notepad for editing this stuff which causes files to be written in ANSI/Windows-1252/ISO-8859-1 instead of UTF-8. Not as if you could not read ANSI files if you put UTF-8, of course.